CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 wrapped, and can be detected by using errors.As/Is practise. This connection
 errors can, and could even before, happen anytime the sql.DB doesn't have free
 connection for executed query.
+* Updated azidentity to 1.2.1, which adds in memory cache for managed credentials
```

Comment:
0.20.0 already went out, add a 0.21.0 entry
We don't have automation of this file content (yet).